### PR TITLE
Improvements to the pw2wannier90 CLI

### PR DESCRIPTION
**Pull request by normarivano**
Some improvements on the pw2wannier CLI. Now  the `single_file` has been renamed into `nnkp_file` (for the sake of clarity). Moreover, the user and can specify the  desired SCDM mode and if UNK matrices are to be written as output.

### CODE REVIEW

File: aiida_quantumespresso/cli/calculations/pw2wannier90.py [hunk 1]
```diff
@@ -16,56 +18,75 @@
...
-            'scdm_entanglement': 'isolated',
         }
     }
 
+    if write_unk:
+        parameters['INPUTPP']['write_unk'] = True
+    else:
+        parameters['INPUTPP']['write_unk'] = False
+
+    if scdm_mode != 'no':
```
Comment (sphuber) [inline aiida_quantumespresso/cli/calculations/pw2wannier90.py]: it doesn't seem that the actual value of `scdm_mode` gets used so why does the option specify those other possible values `'isolated', 'erfc', 'gaussian'`?

File: aiida_quantumespresso/cli/calculations/pw2wannier90.py [hunk 2]
```diff
@@ -16,56 +18,75 @@
...
-            'write_unk': False,
-            'scdm_proj': True,
-            'scdm_entanglement': 'isolated',
         }
     }
 
+    if write_unk:
+        parameters['INPUTPP']['write_unk'] = True
+    else:
+        parameters['INPUTPP']['write_unk'] = False
```
[the remark this answers is not in the document]
Comment (normarivano) [reply aiida_quantumespresso/cli/calculations/pw2wannier90.py]: Thank you!

File: aiida_quantumespresso/cli/calculations/pw2wannier90.py (hunk 1 above)
Comment (normarivano) [reply aiida_quantumespresso/cli/calculations/pw2wannier90.py]: You are right! I've fixed it.

**Comment by sphuber** [comment]
Thanks very much @normarivano

**Pull request merged**